include/rwsdk/rwcore.h

```diff
@@ -72,6 +72,25 @@ struct RwTexture
 	RwInt32 refCount;
 };
 
+struct RwV3d
+{
+	float x;
+	float y;
+	float z;
+};
+
+struct RwMatrixTag
+{
+	RwV3d right;
+	unsigned int flags;
+	RwV3d up;
+	unsigned int pad1;
+	RwV3d at;
+	unsigned int pad2;
+	RwV3d pos;
+	unsigned int pad3;
+};
```

Comment:
These structs should use the `RwReal` and `RwUInt32` types (defined at the top of the file) instead of `float` and `unsigned int`.

In the future I would recommend referencing the official RW SDK when copying types from the PS2 data:
* https://github.com/resigma/rwsdk-v35-ps2
* https://github.com/resigma/rwsrc-v37-pc